src/js/TuningSliders.js

```diff
@@ -241,10 +241,10 @@ TuningSliders.dtermFilterSliderDisable = function() {
 
 TuningSliders.updateSlidersWarning = function(slidersUnavailable = false) {
     const WARNING_P_GAIN = 70;
-    const WARNING_DMAX_GAIN = 60;
+    const WARNING_D_MAX_GAIN = 60;
     const WARNING_I_GAIN = 2.5 * FC.PIDS[0][0];
-    const WARNING_DMIN_GAIN = 42;
-    const enableWarning = FC.PIDS[0][0] > WARNING_P_GAIN || FC.PIDS[0][1] > WARNING_I_GAIN || FC.PIDS[0][2] > WARNING_DMAX_GAIN || FC.ADVANCED_TUNING.dMinRoll > WARNING_DMIN_GAIN;
+    const WARNING_D_GAIN = 42;
+    const enableWarning = FC.PIDS[0][0] > WARNING_P_GAIN || FC.PIDS[0][1] > WARNING_I_GAIN || FC.PIDS[0][2] > WARNING_D_GAIN || FC.ADVANCED_TUNING.dMaxRoll > WARNING_D_MAX_GAIN;
```

Comment:
Perhaps (need to import semver)

```suggestion
        const enableWarning = semver.lt(FC.CONFIG.apiVersion, '1.47.0')
        ? FC.PIDS[0][0] > WARNING_P_GAIN || FC.PIDS[0][1] > WARNING_I_GAIN || FC.PIDS[0][2] > WARNING_D_MAX_GAIN || FC.ADVANCED_TUNING.dMaxRoll > WARNING_D_GAIN
        : FC.PIDS[0][0] > WARNING_P_GAIN || FC.PIDS[0][1] > WARNING_I_GAIN || FC.PIDS[0][2] > WARNING_D_GAIN || FC.ADVANCED_TUNING.dMaxRoll > WARNING_D_MAX_GAIN;
```